### PR TITLE
Activate pane and capture pane

### PR DESCRIPTION
**Pull request by yitsushi**
* Parse PaneID
* Parse ActivePane state
* Ability to call capture-pane
* Add go mod

### CODE REVIEW

**Comment by jubnzv** [comment]
Thank you very much! I will merge it when CI is green. Just one small question.

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module github.com/jubnzv/go-tmux
+
+go 1.15
```
Comment (jubnzv) [inline go.mod]: Does this mean that this module requires at least go 1.15? I used it with go 1.12 and everything worked correctly.

Comment (yitsushi) [reply go.mod]: I can get this to a lower version, it was just an auto generated go mod file.

Comment (yitsushi) [reply go.mod]: done

**Pull request merged**